api/jobs_test.go

```diff
@@ -73,8 +77,8 @@ func TestCreateJobHandlerWithValidID(t *testing.T) {
 func TestGetJobHandler(t *testing.T) {
 	t.Parallel()
 	Convey("Given a Search Reindex Job API that returns specific jobs using their id as a key", t, func() {
-		jobStoreMock := &mock.JobStoreMock{
-			GetJobFunc: func(ctx context.Context, id string, mux *sync.Mutex) (models.Job, error) {
+		jobsCollectionMock := &apiMock.JobStorerMock{
```

Comment:
I'd probably leave this as `jobStoreMock`

Reply:
Not flagged all instances, more below…

Reply:
name reverted back to jobStoreMock now